### DISCUSSION TITLE
I loved the improvement of the Reflection section ❤️

### DISCUSSION THREAD

**Discussion by GrimReaper3223** [java/devjava-content#124] [New Content Ideas]
I am continuing my studies while simultaneously implementing what I learn in my clients' projects ❤️

Then I came across the Reflection section completely built 😃! I thought about looking for the old Java 8 documentation (where there are tutorials) to study about it, because it wasn't available here yet, and I'm so happy to see that the section is finally finished ❤️. Thank you so much

**Comment by JosePaumard** [comment]
Thank you for your kind words, they are very much appreciated!
